libs/dh/metering-point/feature-overview/src/components/dh-metering-point.component.ts

```diff
@@ -134,15 +134,18 @@ import { DhReleaseToggleDirective } from '@energinet-datahub/dh/shared/release-t
               <span
                 gap="s"
                 direction="row"
```

Comment:
nitpick: Unrelated to this PR, so we can fix it later. But these have no effect right? Since it doesn't have a `vater` directive applied.

Reply:
Yeah, you're right. Can see the pattern with other elements in this file. Will fix it in another PR.